artifactory/commands/golang/go.go

```diff
@@ -140,6 +171,66 @@ func (gc *GoCommand) Run() error {
 	return err
 }
 
+// copyGoPackageFiles copies the package files from the go mode cache directory to the given destPath.
+// The path to those chache files is retrived using the supplied package name and Artifactory details.
+func copyGoPackageFiles(destPath, packageName, rtTargetRepo string, rtDetails auth.ServiceDetails) error {
+	packageFilesPath, err := getPackageFilePathFromArtifactory(packageName, rtTargetRepo, rtDetails)
+	if err != nil {
+		return err
+	}
+	// Copy the entire content of the relevant Go pkg directory to the requested destination path.
+	err = fileutils.CopyDir(packageFilesPath, destPath, true, nil)
+	if err != nil {
+		return fmt.Errorf("Couldn't find suitable package files: %s", packageFilesPath)
+	}
+	return nil
+}
+
+// getPackageFilePathFromArtifactory returns a string represents the package files chache path.
```

Comment:
...a string represents the package... --> ...a string that represents the package...